tests/Profile/ProfileElementTest.php

```diff
@@ -47,5 +60,42 @@ public function testValidTypeRestrictions()
         }
     }
 
+    public function testObjectTypeRestrictionsHaveTypes()
```

Comment:
All attributes on object record types, except for those that are used everywhere, are tied down to a record type